exporter/SynthesisFusionGltfExporter/gltf/utils/FusionUtils.py

```diff
@@ -65,23 +65,25 @@ def getPBRSettingsFromAppearance(fusionAppearance: adsk.core.Appearance, exportW
 
     transparent = False
     emissiveColorFactor = None
-    metallicFactor = 0.0
-    roughnessProp = props.itemById("surface_roughness")
-    roughnessFactor = roughnessProp.value if roughnessProp is not None else 0.1
+    metallicFactor = 0.1 # never 0 and certainly never 1
+    roughnessFactor = 0.1
```

Comment:
Added default values to the low end of metallic,

Roughness was modified as a side effect of testing for the exception, no actual change happened except for the null check.